packages/commonlib.js/src/logger.ts

```diff
@@ -18,21 +20,23 @@ export interface BetterLogger extends Logger {
 
 export const createLogger = (
   consoleFormatLogger: Format,
-  logLevel: string
+  logLevel: string,
+  additionnalTransports: Transport[] = []
```

Comment:
Minor type (extra 'n'), should be:

```suggestion
  additionalTransports: Transport[] = []
```

Reply:
Fixed, thanks 👍